### DISCUSSION TITLE
[Tour Type] How to define tour types

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1013] [General]
How do I fill in tour book Type field

**Comment by wolfgang-ch** [comment]
See doku https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-type

**Comment by gringojack** [comment]
How can I apply this:

![2023-03-10 17_22_20-Livre des parcours - MyTourbook - 23 3 0](https://user-images.githubusercontent.com/4082252/224373133-96473013-6cff-4101-a321-09ac20b8b44d.jpg)

To that:

![2023-03-10 17_22_57-Livre des parcours - MyTourbook - 23 3 0](https://user-images.githubusercontent.com/4082252/224373240-03c33ba3-c0aa-4ffa-b043-7ebd6a0646ba.jpg)

thanks

**Reply by Thompson-ongithub** [reply]
You can set the tour type either in the "Tour Editor" or by right-clicking onto the tour in the calendar or the "Tour Book" or even by right-clicking when you import the tour..
